docs/quickstart.rst

```diff
@@ -57,6 +57,8 @@ Settings
       ANVIL_ACCOUNT_LINK_REDIRECT = "home"
       # Specify the subject for AnVIL account verification emails.
       ANVIL_ACCOUNT_LINK_EMAIL_SUBJECT = "Verify your AnVIL account email"
+      # Specify the email to receive notification when account is linked
+      ANVIL_ACCOUNT_VERIFY_NOTIFICATION_EMAIL = "to@example.com"
```

Comment:
This should be a separate section, since it's not a required section. Later on in the docs you could say something like "if you would like to receive emails when a user links their account, set the `ANVIL_ACCOUNT_VERIFY_NOTIFICATION_EMAIL` setting in your settings file." with an example.